src/validateRequest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+
+namespace Marijana\Restpackage;
+
+
+class validateRequest
+{
+    protected string $api_key;
+    protected string $method;
+    protected string $url;
+    protected mixed $param;
+
+    /**
+     * @param $modelName
+     * @param $method
+     * @param $endpointName
+     * @return \Illuminate\Config\Repository|\Illuminate\Contracts\Foundation\Application|mixed
+     */
+    public function readEnd($modelName, $method, $endpointName)
+    {
+       return $point = config('endpoints.'.$modelName.'.'.$method .'.'.$endpointName.'');
+
+    }
+    /**
+     * @param $method
+     * @param $endpointName
+     * @param $param
+     * @return string
+     */
+    public function validate($modelName, $method, $endpointName, $param=null){
+        if($this->getMethod($modelName, $method, $endpointName) == true ){
+           return $this->getUrl($this->getMethod($modelName, $method, $endpointName));
+        }else{
+            return "Bad method and endpointName combination!";
+        }
+    }
+
+    /**
+     * @param $point
+     * @param null $param
+     * @return string
+     * @description check endpoint regularity
+     */
+    public function getUrl($point, $param=null)
+    {
+            if(strpos($point, ':id') !==false && (!empty($param))){
+                $endUrl = str_replace(':id', $param,$point);
+            }elseif(strpos($point, ':id') !==false && (empty($param))){
+                return "Missing param for ".$point;
+            }else{
+                $endUrl = $point;
+            }
+            $baseURL = env('APP_URL');
+            $this->url = $baseURL.$endUrl;
```

Comment:
check indentation for this line and all lines above
code style: `$this->url = $baseURL . $endUrl;`